packages/@okta/vuepress-site/docs/reference/hook-best-practices/index.md

```diff
@@ -0,0 +1,82 @@
+
+>**Note:** This is a large list of IP Addresses and the list is subject to change. Unless required by your organization, securing your Hook by authentication header is recommended.
+
+## Protect your Hook content from external viewers
+
+To prevent unauthorized parties from reading the contents of an Okta Hook, we recommend the following:
+
+* Okta uses HTTPS to encrypt communications to your Hook endpoint. When using HTTPS, ensure you keep your SSL certificate updated and the Domain Name System (DNS) secured, so that someone can’t re-point your endpoint to another location.
+
+## Avoid delays in Hook responses
+
+When Okta uses an Inline Hook to communicate with your endpoint, the user experience is paused until your code responds. Okta Event Hooks, although acting asynchronously, also require a response. To prevent unnecessary delays or timeouts, we recommend the following:
+
+* Treat Okta Hooks as you would any other HTTP request, namely: Ensure you respond to the HTTP request in under 400 milliseconds with a 200 (Success) or 204 (Success no content) code.
+* Process the Hook request data after the response is sent to avoid delays or timeouts.
+
+A timeout of three seconds is enforced on all outbound requests for Event and Inline Hooks, with one retry in the event of a timeout or an error response from the external service. If a successful response has not been received after that, a 400 error is returned with more information about what failed.
+
+See Inline Hook [Timeout and Retry](/docs/concepts/inline-hooks/#timeout-and-retry) for further information on Inline Hook timeout and retry behavior; see Event Hook [Timeout and Retry](https://developer.okta.com/docs/concepts/event-hooks/#timeout-and-retry) for the same.
+
+## Limits, duplicates, and order of Hook calls
+
+The number of Hook calls and the limits per org are available in the following table. Keep in mind these numbers and limits when designing your Hook solution.
+
+The order of Event or Inline Hook calls is not guaranteed. Your external service processing of the Hook requests must consider this for your solution. Also, to avoid processing duplicate requests, use the `eventId` property to identify unique requests.
+
+| Hook Type | Limit Type | Limit | Description |
+| --------- | -----------| ----- | ----------- |
+| Event Hook | Number of daily Event Hooks | 100K | A maximum of 100 thousand Event Hooks can be fired, per org, per day. Event Hooks are not recorded or replayed after this point. If a request times out after three seconds, Event Hooks are retried one time afterward. Retries do not count toward the org limit.
```

Comment:
A maximum of **100,000** Event Hooks can be fired, per org, per day. Event Hooks are not recorded or replayed after **this number is reached**. If a request times out after three seconds, Event Hooks **are retried once**. Retries do not count toward the org limit.